src/models/Factor.js

```diff
@@ -207,7 +207,7 @@ function (Okta, Q, factorUtil, BaseLoginModel) {
         }
 
         // MFA_CHALLENGE
-        else if (this.get('canUseResend') && transaction.resend) {
+        else if (this.get('canUseResend') && !this.get('answer') && transaction.resend) {
```

Comment:
On click `Call` it makes `verify` request.
After we get the code, we fill the `answer` textbox and click `verify`.
Without checking the `answer`, it posts the code with `resend` request, that has error response.
So, to make a `Call` we always have empty `answer` and make the first request with `verify`, others with `resend`.
If we have an `answer`, we always do `verify` request.